guides/RELEASE-PROCESS.md

```diff
@@ -1,55 +1,74 @@
 # Release process
 
 1. **Create a release commit**
-   1. Run prepublish script from the root of your local git repo. This will bump the version, build the documentation, and transpile + bundle `dist` assets.
+
+    1. Get the latest from the `master` branch
+         ```
+        git pull
+         ```
+    1. Check the status of your working directory to see which changes have been staged, which haven’t, and which files aren’t being tracked by Git.
```

Comment:
I would clarify that this is to ensure that you don't have any local changes. We only want them to proceed to the next step if there are no changes